docs/examples/configuration.ipynb

```diff
@@ -1819,9 +1855,22 @@
   }
  ],
  "metadata": {
+  "kernelspec": {
+   "display_name": "Python 3",
+   "language": "python",
+   "name": "python3"
+  },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "pygments_lexer": "ipython3"
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.6.10"
```

Comment:
Do we have a spec for what we do with this metadata? I thought that we were trying to keep it out of notebooks, and I typically use the clear-notebooks script to get rid of it. It really doesn't matter either way, just wondering if we have a policy.

Reply:
Ah yeah I must have just committed this by accident at some point and didn't notice.

Reply:
Ok, undid it.